### ISSUE TITLE
ENOENT .. no discernible stack trace

### ISSUE THREAD

**Issue by seiyria** [arunoda/meteor-up-legacy#139]
```
Meteor Up: Production Quality Meteor Deployments
------------------------------------------------

[127.0.0.1] Warning: Permanently added '127.0.0.1' (ECDSA) to the list of known hosts.
[127.0.0.1] Error: spawn ENOENT
    at errnoException (child_process.js:1001:11)
    at Process.ChildProcess._handle.onexit (child_process.js:792:34)

events.js:72
        throw er; // Unhandled 'error' event
              ^
Error: spawn ENOENT
    at errnoException (child_process.js:1001:11)
    at Process.ChildProcess._handle.onexit (child_process.js:792:34)
```

**Comment by arunoda** [comment]
can you please check the `appPath` in `mup.json`.
seems like it's not correct.

**Comment by seiyria** [comment]
I don't appear to have an appPath?

```
{
  // Server authentication info
  "servers": [
    {
      "host": "127.0.0.1",
      "username": "root",
      "password": "mypassword"
      // or pem file (ssh based authentication)
      //"pem": "~/.ssh/id_rsa"
    }
  ],

  // Install MongoDB in the server, does not destroy local MongoDB on future setup
  "setupMongo": false,

  // WARNING: Node.js is required! Only skip if you already have Node.js installed on server.
  "setupNode": false,

  // WARNING: If nodeVersion omitted will setup 0.10.31 by default. Do not use v, only version number.
  "nodeVersion": "0.11.13",

  // Install PhantomJS in the server
  "setupPhantom": false,

  // Application name (No spaces)
  "appName": "kurea",

  // Location of app (local directory)
  "app": "/root/kurea.web/",

  // Configure environment
  "env": {
    "ROOT_URL": "127.0.0.1",
    "PORT": 80,
    "MONGO_URL": "mongodb://localhost:27017/kurea"
  },

  // Meteor Up checks if the app comes online just after the deployment
  // before mup checks that, it will wait for no. of seconds configured below
  "deployCheckWaitTime": 45
}
```

**Comment by arunoda** [comment]
appPath needs to be the path on your local machine.

---

Arunoda Susiripala

@arunoda http://twitter.com/arunoda
http://gplus.to/arunodahttps://github.com/arunoda
http://www.linkedin.com/in/arunoda

**Comment by seiyria** [comment]
So would that be app then? Or is app used for something else?

**Comment by arunoda** [comment]
it should point to the meteor app on your local dev box.

---

Arunoda Susiripala

@arunoda http://twitter.com/arunoda
http://gplus.to/arunodahttps://github.com/arunoda
http://www.linkedin.com/in/arunoda

**Comment by seiyria** [comment]
Well, `mup init` or whatever generated it with app instead of appPath, but now I'm getting a different error. I'll close this issue though.

**Comment by seiyria** [comment]
Here's my error now, though: http://pastebin.com/uPNvSKam For reference, I'm getting a similar issue to #46 (curl failed to check). I did watch netstat and ps aux | grep meteor, I did not see port 80 come up in netstat but meteor did start running, so I figure something else is at play. Is there any debugging information that I could provide that would help? Since that stack trace is pretty unhelpful.
